spec/acceptance/fixtures/instance.pp.tmpl

```diff
@@ -9,6 +9,10 @@ ec2_instance { '{{name}}':
     {{k}} => '{{v}}',
   {{/tags}}
   },
+  block_devices => [{
+    device_name => '{{device_name}}',
+    volume_size => {{volume_size}},
```

Comment:
Ah, it was also used in the loadbalancer acceptance test and I'd missed one of the config hashes. Should be fixed now.